core/serialize/serialize.h

```diff
@@ -6,7 +6,7 @@
 namespace sorbet::core::serialize {
 class Serializer {
 public:
-    static const u4 VERSION = 4;
+    static const u4 VERSION = 5;
```

Comment:
We changed the format of serialization (files include file hashes now, which adds a bit even if no file hashes are present), so I think the correct thing to do was to bump the VERSION.

Reply:
@DarkDimius idk what this VERSION constant means

Reply:
See comment below RE:store-state for more context.